src/components/AppointmentConfigModal/DurationInput.vue

```diff
@@ -53,31 +53,60 @@ export default {
 	data() {
 		return {
 			id: randomId(),
+			internalValue: '',
 		}
 	},
 	computed: {
 		valueInMinutes() {
+			// Convert value prop from seconds to minutes
 			return Math.round(this.value / 60)
 		},
+		valueWithUnit() {
+			return this.$t('calendar', '{duration} minutes', {
+				duration: this.valueInMinutes,
+			})
+		},
+		parsedInternalValue() {
+			const matches = this.internalValue.match(/([0-9]*)\s*(minutes)?/i)
```

Comment:
Right. My brain went afk for a moment there :exploding_head:

Reply:
Soo we basically have to drop rtl to make this work, right? Or is there a better way than just replacing `minutes` with a translated string?

Reply:
Only thing I can think of is basic search and replacing the translated string.

Reply:
I thought about it and decided to drop the search for a string completely. A regex lile `/[0-9]+/` should suffice because we only care about the number.